README.md

```diff
@@ -17,7 +19,7 @@ You can save a copy of the file in your personal google drive to make edits and
 
 Plots created in notebooks 1-4 can be viewed without running the code in the "OSW and Solar Analysis Results.ppt". 
 
-Github and notebooks 1-5 created by Jack Ruzekowicz, UCS summer 2024 fellow. Contact: jruzekowicz@gmail.com
+Github and notebooks 1-5 created by Jack Ruzekowicz, UCS summer 2024 fellow from July-September. Contact: jruzekowicz@gmail.com
```

Comment:
I would add a section here (but also, since it's on GitHub, all of the credits will be stored 😄 ). 

```suggestion
## Credits

Notebooks 1-5 were created by Jack Ruzekowicz, UCS Summer 2024. 
Contact: jruzekowicz@gmail.com
```

Reply:
@jruzekow, note the "commit suggestion" button. You can directly adopt the suggestion without doing it yourself.